ci-operator/step-registry/workers-rhel/upgrade/run/workers-rhel-upgrade-run-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: workers-rhel-upgrade-run
+  from: ansible
```

Comment:
If you use `cli` here, there is no need to curl the openshift-client-linux.tar.gz above.

Reply:
ah, got your point. In the step we use some ansible command like ansible-inventory to upgrade rhel nodes. Are they included in the `cli` image?